sarahbenmessaoud.svg

```diff
@@ -0,0 +1,106 @@
+<svg viewBox="0 -25 400 200" xmlns="http://www.w3.org/2000/svg">
```

Comment:
Remove this file from this PR, it’s uploaded to the root of our project, and doesn’t belong in your bar chart!